### DISCUSSION TITLE
Can we open this project for hacktoberfest?

### DISCUSSION THREAD

**Discussion by VikalpRusia** [rancher-sandbox/rancher-desktop#7554] [General]
This will gather more support for this project.
Link - https://hacktoberfest.com/participation/#maintainers

**Comment by mook-as** [comment]
Hacktoberfest has historically been overrun with low-effort contributions, ever since that one guy make a video about how you can get a free shirt. (I realize that they don't have any physical goods anymore.) We currently do not have the people necessary to manage the horde that would be expected to show up if we opt-in.

At this time I don't think we can opt-in.
